### PR TITLE
Make `times` parameter doc more clear.

### PR DESCRIPTION
**Pull request by ethan256**
update trigger docs [#8589](https://github.com/apache/skywalking/issues/8589)

### CODE REVIEW

File: internal/components/trigger/http.go [hunk 1]
```diff
@@ -79,7 +79,8 @@ func (h *httpAction) Do() chan error {
 			case <-t.C:
 				err := h.execute()
 
-				// `h.times == h.executedCount` makes sure to only send the first error
+				// `send == false && h.times == h.executedCount` makes sure to only send firstly the error and
```
Comment (wu-sheng) [inline internal/components/trigger/http.go:82]: What is the `send firstly the error`? Is this?

```suggestion
				// `send == false && h.times == h.executedCount` makes sure to only send the error firstly and
```

File: internal/components/trigger/http.go [hunk 2]
```diff
@@ -79,7 +79,8 @@ func (h *httpAction) Do() chan error {
...
+				// ignore errors before.
```
Comment (kezhenxu94) [inline internal/components/trigger/http.go:83]:
```suggestion
				// `err == nil`: if no error occurs, everything is OK and send `nil` to the channel to continue.
				// `h.times == h.executedCount`: reach to the maximum retry count and send the `err`, no matter it's `nil` or not.
```

**Comment by kezhenxu94** [comment]
@ethan256 updated the comment, hope it's more clear for you

File: docs/en/concepts-and-designs/module-design.md [hunk 3]
```diff
@@ -66,7 +66,7 @@ Generate traffic by trigger the action, It could access `HTTP API` or execute `c
 
 It could have these settings:
 1. **interval**: How frequency to trigger the action.
-1. **times**: How many times to trigger the action, `0=infinite`.
+1. **times**: how many times the operation is triggered before aborting on the condition that trigger had failed always. `0=infinite`.
```
Comment (wu-sheng) [inline docs/en/concepts-and-designs/module-design.md:69]:
```suggestion
1. **times**: How many times the operation is triggered before aborting on the condition that the trigger had failed always. `0=infinite`.
```

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**